<h1 style="margin-left:0px;">
    <span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>AI Methods for the EXHYTE Cycle</strong></span>
</h1>
<p style="margin-left:0pt;text-align:justify;">
    <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">To construct the survey corpus, we focused on the literature published between 2018 and July 2025 to capture recent advances in LLMs and agent systems for scientific discovery. Articles were retrieved through keyword searches on the arXiv and Semantic Scholar APIs, using queries such as LLM, idea generation, hypothesis generation, scientific discovery, experiment design, material discovery, molecular design, drug discovery, and biomedical discovery. Retrieved results were then manually filtered to remove irrelevant articles and ensure inclusion of high-quality papers from major conferences and journals. The final corpus comprises 83 papers spanning diverse disciplines.&nbsp;</span>
</p>
<p style="margin-left:0pt;text-align:justify;">
    <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark12"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">The publication volume in this area surged between 2023 and 2024 and continued to rise sharply in 2025 (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark12"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">A), reflecting both growing cross-domain interest and rapid methodological progress in applying AI to scientific discovery. In addition, across subject areas, most papers come from Applied Sciences and Engineering, followed by Social, Chemical, and Biological Sciences (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark12"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">B). The distribution of highly cited papers over time (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark12"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">D) reveals that several works have gained meaningful impact within the research community. Based on our systematic review of the corpus, 49 papers progress beyond exploration to the HYPOTHESIZE stage(idea + hypothesis generation), yet only 16 continue to the TEST stage (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">C). This disparity highlights a persistent disconnect between hypothesis generation and empirical verification in contemporary AI-assisted scientific discovery. Together, these panels highlight both the rapid growth of the field and the increasing influence of certain works.</span>
</p>
<p style="margin-left:0pt;text-align:justify;">
    <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">To systematically analyze and organize this rapidly expanding body of work, we identified recurring AI strategies that address challenges in each substage of the EXHYTE cycle. Organizing the literature around these strategies moves beyond task- or domain-specific taxonomies, providing a structured view of how AI methods operate across the EXHYTE workflow. In the following sections, we present these strategies substage by substage, defining core patterns, summarizing representative studies and domain-specific variants, and highlighting how each approach is implemented and supported by empirical evidence.</span>
</p>
<ol style="-webkit-text-stroke-width:0px;caret-color:rgb(0, 0, 0);color:rgb(0, 0, 0);counter-reset:c1 1;font-family:-webkit-standard;font-style:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;margin-bottom:0px;margin-right:0px;margin-top:0px;orphans:auto;padding:0px 0px 0px 0pt;text-align:start;text-decoration:none;text-indent:0px;text-transform:none;white-space:normal;widows:auto;word-spacing:0px;" id="l1">
    <li style="display:block;margin:0px;padding:0px;text-indent:0px;" data-list-item-id="e0c06a38052626d8772047a572c6f1305" data-list-text="1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 16pt;text-align:justify;text-decoration:none;text-indent:14pt;">
            &nbsp;
        </p>
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 16pt;text-align:justify;text-decoration:none;text-indent:14pt;">
            &nbsp;
        </p>
    </li>
</ol>
<figure class="image" style="text-align: center; margin-top: 20px; margin-bottom: 20px;">
    <picture><img src="image001.png" width="550" height="550" style="max-width: 100%; height: auto;"></picture>
</figure>
<p class="s1" style="color:black; font-family:'Times New Roman', serif; font-size:10pt; font-style:normal; margin:0pt; padding:6pt 0px 0px 0px; text-align:center; text-decoration:none; text-indent:0pt; line-height: 1.4;">
    <a name="bookmark12"></a>
    
    <span style="font-family:'Times New Roman', Times, serif; font-size:medium;">
        <strong>Fig. 1 Temporal and thematic analysis of publications.</strong>
    </span>

    <span style="color:black; font-family:'Times New Roman', Times, serif; font-size:medium;">
        (A) Quarterly count of publications over time. The bar height represents the number of papers published in each quarter. 
        (B) Quarterly distribution of publications by subject area. Bars indicate the number of papers in each subject area per quarter. 
        (C) Upset plot showing the number of publications covering different stages of the EXHYTE cycle (Explore, Idea, Hypothesis, Test). Bars represent the number of papers in each combination of stages. 
        (D) Scatter plot of highly cited papers, with citations per month on the y-axis and quarter/year on the x-axis. The overlaid bar plot indicates the number of highly cited papers per quarter.
    </span>
</p>
<br>
<ol style="counter-reset:c2 1;list-style-type:decimal;margin-bottom:0px;margin-right:0px;margin-top:0px;padding:0px 0px 0px 0pt;text-indent:0px;" id="l2">
    <li class="ck-list-marker-bold ck-list-marker-font-size ck-list-marker-font-family" style="--ck-content-list-marker-font-family:'Times New Roman', Times, serif;--ck-content-list-marker-font-size:22px;display:block;margin:0px;padding:0px;text-indent:0px;" data-list-item-id="e77068b42b0f88e6376c2dd66389e3912" data-list-text="1.1">
        <p style="margin-left:0px;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark1"><span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>‌</strong></span></a><span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>Inputs to the EXHYTE Cycle</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">In automated scientific discovery systems, user input ranges from natural language queries about research topics to domain-specific research papers or datasets, playing a central role in aligning system behavior with specific goals. High-level research directions, such as workshop themes, set the scope for the automated scientific workflow. For example, the ICLR 2025 workshop “I Can’t Believe It’s Not Better” (ICBINB) focused on real-world pitfalls, challenges, and negative or inconclusive results in deep learning, a perspective incorporated in AI-Scientist-v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]. Similarly, focused problem statements, such as "How can we improve the energy efficiency of AI models?" (e.g., SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark20"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), guide idea and hypothesis generation. Extending this idea, IRIS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] formalizes user input as research goals that explicitly decompose into a problem and its motivation/context. For example, the research goal “To improve the balance between quality and diversity in text generation (Problem) by addressing the limitations of existing sampling methods like top-p (nucleus sampling), which often </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark21"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">produce incoherent or repetitive outputs at higher temperatures (motivation/context)”. Structured, domain-specific inputs similarly allow users to express goals and constraints in scientific formats. For example, LLMatDesign [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">4</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark22"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] begins with user-specified chemical compositions and target properties for candidate materials, while AutoMAT [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">5</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark23"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] accepts user-specified targets such as yield strength, density, or elemental cost constraints. In drug discovery, systems such as [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">6</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark24"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] operate on query molecules represented as SMILES strings and PiFlow [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">7</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] integrates foundational scientific laws or expert-defined principles to guide search.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Beyond explicit goal specification, user-provided resources such as research papers and datasets function as empirical and contextual inputs. Systems like ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], CodeScientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] leverage literature retrieval and idea grounding. CodeScientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], for example, summarizes vetted code blocks to ensure that gener- ated research ideas are practical, implementable, and executable. Similarly, datasets provide an empirical foundation for discovery. DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark29"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], for example, employs molecule–protein interaction datasets such as DrugBank [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">12</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark30"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and KIBA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">13</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] to support drug discovery tasks.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Together, these examples highlight the pivotal role of user input, whether conveyed through language, structured scientific specifications, or empirical data, in steering automated discovery workflows across diverse scientific domains.</span>
        </p>
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 16pt;text-align:justify;text-decoration:none;text-indent:14pt;">
            &nbsp;
        </p>
<figure class="image" style="text-align: center; margin-top: 20px; margin-bottom: 20px;">
    <img src="image002.jpg" width="550" height="550" style="max-width: 100%; height: auto;">
</figure>
    </li>
</ol>
<p class="s1" style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:8pt;font-style:normal;margin:0pt;padding:0px 0px 0px 40pt;text-align:center;text-decoration:none;text-indent:0pt;">
    <span style="font-family:'Times New Roman', Times, serif;font-size:medium"><strong>Fig. 2 </strong></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:8pt;"><span class="s2" style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;font-size:medium"><strong>Inputs and outputs associated with each substage of the EXHYTE workflow (Explore, Hypothesize, Test).</strong></span></span>
</p>
<ol style="counter-reset:c2 1;list-style-type:decimal;margin-bottom:0px;margin-right:0px;margin-top:0px;padding:0px 0px 0px 0pt;text-indent:0px;" id="l2">
    <li style="display:block;margin:0px;padding:0px;text-indent:0px;" data-list-item-id="eb158e75e9a74c524c87b7a45da53a263" data-list-text="1.1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-align:left;text-decoration:none;text-indent:0pt;">
            &nbsp;
        </p>
    </li>
    <li class="ck-list-marker-bold ck-list-marker-font-size ck-list-marker-font-family" style="--ck-content-list-marker-font-family:'Times New Roman', Times, serif;--ck-content-list-marker-font-size:22px;display:block;margin:0px;padding:0px;text-indent:0px;" data-list-item-id="ecbb8c32291e836215b41f35ecd698935" data-list-text="1.2">
        <p style="margin-left:0px;text-align:justify;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark2"><span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>‌</strong></span></a><span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>The Explore Stage</strong></span>
        </p>
        <p style="margin-left:0px;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E1: Query Structuring </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">User query structuring is the first step in the Explore Stage, where it transforms raw, unstructured research inputs into actionable representations. This process employs complementary strategies including decomposition, vector representation, and information extraction from datasets to enable targeted exploration across literature and data (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">A and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0px;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E1-S1: Decomposition. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Decomposition transforms complex research queries into tractable sub-queries/questions or tasks that enable focused literature/dataset retrieval, reasoning, and solution design using LLM. Different systems operationalize decomposi- tion at various levels. Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] decomposes initial research topic into multiple sub-queries that capture diverse perspectives on the same problem., SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark32"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extracts key-phrases that represent the underlying research focus of a scientist’s query, Assay2Mol [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">15</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] identifies keywords from user-provided text descriptions of target pro- teins or phenotypes, Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark34"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] formulates a series of general questions based on the name of the target disease that the scientist inputs into the system, DrugLLM [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">17</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] decomposes molecules into structural groups to facilitate reasoning about chemical properties.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Building on this principle, several systems employ LLM-based agents to decom- pose complex user inputs—whether research queries or design tasks—into smaller, manageable sub-tasks and to generate detailed, step-by-step workflow plans. General- purpose frameworks such as ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AI-Researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark36"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">19</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark37"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Zhou et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">20</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark38"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and domain specific system such as ProtAgent(protein design) [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">21</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">],</span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark39"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">AtomAgents(alloy design) [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">22</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], MatExpert(material discovery) [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark41"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">],DrugAgents(drug discovery) [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">24</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] all follow similar decomposition pipeline.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E1-S2: Vector representations. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Encoding user inputs as vector representations enables efficient retrieval, semantic similarity search, and context-aware reasoning. Frameworks such as Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark20"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], IRIS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] transform research queries, questions, background knowledge into embeddings to support semantic retrieval of relevant scientific literature.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark44"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">In domain-specific applications, Chemist-X [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">27</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] encodes molecules as SMILES strings to retrieve reaction conditions for target products, while DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark32"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] identifies structurally similar molecules using vectorized molecular representations. Assay2Mol [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">15</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] embeds protein target descriptions and phenotypic data for BioAssay retrieval, and MatExpert [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employs dual encoders with one for property descriptions and another for structural representations to improve reference material retrieval. Collectively, these systems underscore the pivotal role of vector representations in </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:5pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]enabling semantically rich, machine-interpretable forms of knowledge that drive efficient and context-aware scientific discovery</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>E1-S3: Information Extraction from Datasets. </strong></b></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Extracting key features and annotations from structured datasets enables automated systems to generate empirical inputs for downstream tasks such as model training, hypothesis formulation, and idea generation. AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], for instance, employs a <i style="margin:0px;padding:0px;text-indent:0px;">Data Analyst Agent </i></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark46"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">to identify polycyclic aromatic hydrocarbon (PAH) patterns from user-provided mass spectrometry data, thereby generating testable hypotheses in astrobiology. Proteus [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">29</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark46"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] integrates bioinformatics tools to uncover biological relationships through statistical analyses, while Qu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">29</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] further identify pairs of biological entities (e.g., protein, clinic) to define research directions from multi-omics datasets. ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieves </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">protein sequence descriptions, structural features, and physicochemical properties to inform LLMs about sequence–structure–function relationships during hypothesis generation. In drug discovery, DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark29"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extracts molecular properties and annotations from DrugBank [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">12</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark30"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and KIBA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">13</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], while MatExpert [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark48"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extracts chemical formulas and physical properties (e.g., formation energy, elemental composition) from databases such as NOMAD[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">31</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark49"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and Materials Project[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">32</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark50"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] to generate candidate crystal structures. LLM-Feynman [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">33</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] automates feature selection and iterative refinement to construct high-quality feature sets, which are subsequently used in symbolic regression to derive interpretable physical equations. Together, these systems illustrate how information extraction transforms raw datasets into structured, semantically meaningful inputs that drive empirical reasoning and hypothesis development across scientific domains.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E2: Data Retrieval </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Retrieving literature and datasets is a foundational step in scientific discovery, ensuring that exploration remains grounded in comprehensive and relevant knowledge. Guided by user inputs, this E2 step compiles scientific publications and datasets that support informed hypothesis exploration and validation. We identified three key strategies that capture how systems retrieve information through LLM-based agents, related-work networks, and scientific repositories, respectively (Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">A and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E2-S1: LLM Agent–Based Retrieval. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">LLM agents now routinely query and curate literature and datasets. General-purpose frameworks such as Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark51"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and Graph of AI Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">34</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark36"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] use LLMs to generate multiple research queries from an initial topic, capturing diverse conceptual angles. SciAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">19</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark52"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and AI-Scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">35</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], along with AI-Scientist-v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">], retrieve literature via the Semantic Scholar API for novelty checks and evidence gathering. </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark20"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">LLM agents are also tasked with prioritizing the most relevant papers. IRIS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark53"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] introduces a retrieval agent that issues targeted queries answered by the specialized “AI2 Scholar QA” API. Moose-Chem [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">36</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] generates titles from background papers to retrieve inspirational works. AI-Researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] designs a “Knowledge Acquisition Agent” that, given reference papers, retrieves additional papers and code repositories from scientific databases. In domain-specific settings, retrieval is coupled with scientific reasoning. DrugAgent </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark41"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">24</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark39"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employs an “Instructor Agent” to query domain libraries and translate abstract ideas into executable code via domain-specific examples. AtomAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">22</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark44"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] integrates a specialized knowledge-retrieval tool that extracts material properties directly from published papers. Chemist-X [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">27</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark38"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] produces executable code to interrogate repositories such as PubChem for chemical literature and molecular datasets, while ProtAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">21</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark22"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieves specific data points (e.g., Protein Data Bank (PDB) identifiers) for protein design. In materials discovery, AutoMAT [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">5</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses LLMs, together with user-defined property targets and constraints, to consult reference handbooks (e.g., titanium alloy databases) and identify candidates meeting elemental criteria.&nbsp;</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E2-S2: Related-Work Based Retrieval. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Citation networks and semantic similarity are widely used to link related works and surface new directions. ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">], </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] expand from an anchor paper by retrieving both citing and cited works to capture the evolution of research themes; HypER further adds LLM-based relevancy scoring. AI-Researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] incorporates multi-step reference selection for each target paper, including citation-pattern analysis (frequency), context analysis (influence on methods/theory/experiments), evidence collection (textual support), and an integrated impact score. SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] combines keyword, co-occurrence, and semantics-based retrieval, then applies a cluster-based selection of representative papers. SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark54"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] queries academic sources using key phrases drawn from a researcher’s work and similar literature. Scideator [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">37</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieves multiple analogs per source paper, uses an LLM to generate “purpose–mechanism” pairs, and then issues queries per pair to find additional relevant papers. Domain-focused approaches tailor queries to specialized terms. For example, ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses domain-specific keywords (e.g., “Protein Science”) to retrieve metadata (titles, abstracts, links); AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] selects relevant papers and books to provide astrobiology context; and Tong et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark56"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] require the presence of terms such as “psychol,” “clin psychol,” and “biol psychol” in titles or abstracts for psychology/neuroscience. Manual and semi-automatic curation complements these methods via direct selection from leading databases (e.g., Sparks of Science [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">39</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Perovskite-R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Budget AI Researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark23"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Lee et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">6</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark59"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Quilodrán-Casas et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">42</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Cheng et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark61"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Ciucua et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">44</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E2-S3: Dataset Retrieval from Scientific Repositories. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Datasets provide the empirical foundation for analysis, model training, and validation. In drug discovery, DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] queries databases to retrieve molecules structurally similar to a query compound, associated protein interactors, and binding-pocket information parsed from Protein Data Bank (PDB) files to improve LLM interpretability. DrugAgent </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark41"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">24</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark62"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieves biological datasets from the Therapeutics Data Commons (TDC), which offers structured, AI-ready datasets and benchmarks across drug development stages. In materials discovery, FlowLLM [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">45</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieves inorganic crystalline materials from the Materials Project for LLM fine-tuning, and Perovskite-R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] assembles drug-like compounds from a comprehensive library to enhance generalization across chemical space. MatExpert [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark21"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and LLMatDesign [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">4</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark48"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] access NOMAD [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">31</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark49"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and the Materials Project [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">32</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark37"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] for reference materials used in training, evaluation, and modification workflows. Zhou et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">20</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieve crystal structures with compositions or reduced formulas similar to a query from structural databases to identify promising crystal prototypes.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E3: Knowledge Assembly </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">The knowledge-assembly stage transforms unstructured scientific text into structured, queryable representations that enable efficient reasoning during discovery. The existing literature typically employs three strategies: structured extraction and summarization from literature, knowledge-graph construction, and database construction for large-scale retrieval and synthesis(Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">A and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E3-S1: Structured extraction and summarization from literature </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">To make literature readily usable for idea and hypothesis generation, systems extract and summarize standardized sections (e.g., titles, abstracts, methods, results). SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">], </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] implement pipelines that segment papers into consistent fields. Papers like SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark39"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AtomAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">22</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark63"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Literature Meets Data [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">46</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark64"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AI-co-scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">47</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark20"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and IRIS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark56"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] further produce concise synopses of key findings. Sparks of Science [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">39</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses LLMs to extract each paper’s conventional assumption and innovative approach alongside a concise insight summary from abstracts. Several systems including SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark54"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Scideator [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">37</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark65"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Chen et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">48</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extract facets such as objectives, methods, evaluation, and future work. Domain-tailored extraction includes causal relations in psychology (Tong et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), question–answer pairs for Perovskite precursors (Perovskite-R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), frequent-topic mining to guide retrieval (Lee et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark66"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), structured tables of target properties, related structures, and linking mechanisms (Beyond Designer’s Knowledge [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">49</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark22"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), parsing mechanical properties and compositions to propose alloys (AutoMAT [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">5</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark67"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]), and sentence-level evidence selection for property-optimization queries (dziner [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">50</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]).&nbsp;</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>E3-S2: Knowledge graph construction. </strong></b></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Knowledge graphs (KGs) encode entities and relations to support structured reasoning and discovery. SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] represents tasks, methods, materials, and metrics as nodes, with “used-for” edges to ground gen- erated ideas. Tong et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark68"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] construct causal KGs in psychology, where (cause, effect) pairs are directed nodes with attributes (interpretations) and relation types (causal vs. correlational). Y-Mol [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">51</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] builds a biomedical KG with interactions among drugs, genes, and diseases (Drug–Target, Drug–Drug, and Disease–Gene). ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] assem- bles an entity-centric store from co-occurrences in literature to expose cross-domain connections (topics, keywords, individuals, events). SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark51"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] links papers to LLM- extracted keywords in a paper–keyword graph for efficient keyword-based retrieval. Graph of AI Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">34</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] constructs a paper-level graph using structured quadruples <i style="margin:0px;padding:0px;text-indent:0px;">(paper, citation position, citation semantics, paper)</i>, where citation position captures reference importance and citation semantics defines the purpose of each citation.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>E3-S3: Database construction. </strong></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">Curated databases consolidate heterogeneous resources into structured repositories for retrieval, analysis, and synthesis. SciPIP</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] builds a literature database from top venues over the past decade, storing key sections and re-summarizing each paper into a quintuple (keywords, background, ideas, concise methods, core references), with background/ideas encoded as embeddings. ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extracts entities from titles/abstracts (e.g., post-2023 Semantic Scholar) and encodes their co-occurrences in a sparse matrix. HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] compiles temporal reasoning-chain datasets linked to related papers for effective reasoning over scientific literature. SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark56"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] builds corpora from the ACL Anthology and PubMed, organizing the papers with extracted (Background, Target) pairs, seed terms, and their relations. Sparks of Science [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">39</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] constructs the “HypoGen” dataset by extract- ing assumptions, innovations, and core insights from abstracts. ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Lee et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Cheng et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] store vector embeddings of papers from arXiv and major conferences (e.g., NeurIPS, ICLR) to support scalable retrieval.</span>
        </p>
<ol style="counter-reset:c3 1;margin-bottom:0px;margin-right:0px;margin-top:0px;padding:0px 0px 0px 0pt;text-indent:0px;" id="l3">
    <li class="ck-list-marker-font-size ck-list-marker-font-family" style="--ck-content-list-marker-font-family:'Times New Roman', Times, serif;--ck-content-list-marker-font-size:18px;display:block;margin:0px;padding:0px;text-indent:0px;" data-list-item-id="e184e20966e4a526968c0525a69dfff47" data-list-text="1.2.1">
        <p style="margin-left:0pt;text-align:justify;">
            &nbsp;
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>The Hypothesize Stage</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H1: Hypothesis/Idea Generation </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Hypothesis and idea generation mark the transition from structured knowledge to novel discovery, synthesizing literature, data, and reasoning strategies into creative, testable hypotheses. We identify four strategies commonly employed in the literature, including zero-shot generation, multi-agent generation, literature-based generation, and data-driven generation(Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">B and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H1-S1: Zero-shot generation. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark21"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Zero-shot generation relies on the internal knowledge encoded in LLMs, thus enabling idea/hypothesis creation without explicit external context—useful when domain-specific training data are limited. Frameworks illustrating this approach include LLMatDesign [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">4</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark69"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Buehler et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">52</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark70"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Tang et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">53</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark71"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Yin et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">54</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark72"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Bystronski et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">55</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark73"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Abdel et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">56</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark74"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Rabby et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">57</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark75"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Guo et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">58</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark24"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Pu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">7</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark76"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Liu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">59</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark77"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Manning et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">60</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">].</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H1-S2: Multi-agent generation. </strong></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">Multi-agent frameworks build specialized agents, coordinated under an orchestrator agent to produce and refine hypotheses. AstroAgents </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark36"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employs a hierarchical design, where a ‘Data Analysis Agent’ first uncovers key patterns and trends and a ‘Planner Agent’ delegates distinct data segments to multiple ‘Scientist Agents’ for in-depth exploration and investigation; finally each scientist operates within its assigned domain to produce novel hypotheses, which are consolidated by an ’Accumulator Agent’ that deduplicates and integrates the results. In the KG-grounded SciAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">19</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], an Assistant Agent drafts an initial subgraph to contextualize the idea, an Ontologist Agent defines nodes and interprets edges, and two Scientist Agents iteratively draft, elaborate, and critique detailed hypotheses. Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] specializes in biomedicine and designed an agent ’Crow’ to identify causal disease mechanisms and suitable <i style="margin:0px;padding:0px;text-indent:0px;">in vitro </i></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">models and another agent ’Falcon’ to conduct deep literature review to evaluate therapeutic candidates. In chemical discovery, DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] coordinates multiple sequential agents to incrementally build reasoning paths for protein target identification, where a ’Molecule-Analysis’ Agent produces natural-language molecular reports, a ’Molecule-Selection Agent’ filters candidates, an ’Interaction-Analysis’ Agent explores molecule–protein interactions, and a ’Decision’ Agent synthesizes upstream findings to predict promising targets.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H1-S3: Literature-based generation. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Literature-driven approaches retrieve, syn- thesize, and recombine knowledge from scientific texts to propose novel hypotheses. ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark53"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and Moose-Chem [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">36</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] provide titles/abstracts of various related papers as context for LLM. Genesys [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark67"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] formulates new LLM design ideas by modi- fying parent designs from the Knowledge Engine (KE), which integrates the curated literature with code and external sources. Similarly, dziner [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">50</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark64"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] generates modifications based on retrieved material-design guidelines mined from unstructured text (e.g. scien- tific literature and the Internet). Another common strategy is summarization-based synthesis, where relevant sections are distilled and restructured before prompting the LLM (e.g., AI-co-scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">47</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Perovskite-R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">]). Another approach is facet recombination, in which facets (e.g. purpose and mech- anism) are recombined to surface gaps and generate ideas. For example, Scideator</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark54"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">37</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] constructs analogies from purpose–mechanism pairs across papers; SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark66"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] analyzes similar facets to detect patterns, inconsistencies, and unexplored areas; Liu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">49</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and Lee et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] synthesize novel interdependencies among mechanisms or topics not explicitly linked in the source literature; and CODESCIENTIST [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] explores combinatorial idea generation via genetic operators (e.g., crossover, mutation). A com- plementary line of work is a reasoning-chain–based generation, which reconstructs explanatory pathways from the noisy/fragmented literature. HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark56"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Sparks of Science [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">39</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark51"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Graph of AI Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">34</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] fall into this line and identify valid chains in the literature graphs and use them as scaffolds for evidence-grounded hypotheses. Finally, KGs are proposed to structure scientific concepts and relations for systematic hypothesis generation. For example, Tong et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark23"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employ link prediction algorithms on causal knowledge graphs to identify high-probability causal relationships among previously unconnected concepts.; Lee et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">6</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark78"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] integrate biochemical drug–drug rela- tionships from a KG to interpret the network structure, then pass these summaries to a prediction agent; Gao et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">61</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] generate plausible logical hypotheses from KG entity sets.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H1-S4: Data-driven generation. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Structured datasets provide a complementary source, where data instances and patterns can guide hypothesis formation. One group of systems detects patterns and supplies them as context to the LLM. In AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], a ’Data Analyst’ Agent examines mass spectrometry to uncover PAH(Polycyclic Aromatic Hydrocarbons) distributions and alkylation patterns, highlighting unex- pected findings. ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark46"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] analyzes sequence motifs, secondary-structure correlations, and functional-site patterns from experimental data. Proteus [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">29</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark79"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] runs targeted statistical analyses over multi-omics data using bioinformatics tools, recording quantified relationships between biological entities (e.g. proteins, transcripts, variants) and clinical features (e.g. survival time, tumor stage) as edges in a ’Conclusion Graph’ that then grounds LLM hypothesis generation. Another line seeds the LLM with few- shot data instances: HypoGeniC [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">62</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark63"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Literature Meets Data [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">46</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark80"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and Hypotheses for Inductive Reasoning [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">63</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark81"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] provide small sets of (input, label) pairs to induce initial hypotheses. In DrSR [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">64</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark82"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], ChatSR [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">65</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark50"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], and LLM-Feynman [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">33</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark83"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], observational pairs guide LLM in analyzing structural relationships, which are then used by symbolic regression to produce interpretable formulas. Zheng et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">66</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark84"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] discover novel molecular patterns using structural/chemical features and then use them to predict molecular properties. Liu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">67</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark37"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] retrieve domain-specific drug data as demonstrations for LLM to perform drug-editing tasks. In material, MAPPS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">20</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] queries structural databases for crystal prototypes with compositions/reduced formulas similar to a query, using them to generate initial candidate structures; MatExpert [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark85"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] fine-tunes a small LLM on curated transition pathways and target materials from NOMAD to generate plausible transitions pathways. In healthcare and clinical design, Zhu et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">68</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] and Bani et al. </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark86"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">69</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] fine-tune LLMs on patent–title pairs or clinical datasets containing various clinical scenario and ground-truth diagnoses to enable new design ideas and hypotheses.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H2: Hypothesis or Idea Prioritization </strong>Hypothesis prioritization focuses analytical and computational resources on the most promising ideas by evaluating their novelty, plausibility, feasibility, and potential </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">impact. We identify four common strategies: LLM-based evaluation, literature-based assessment, quantitative assessment using domain metrics, and human evaluation(Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">B and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H2-S1: LLM-based evaluation. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">LLM agents can act as evaluators, ranking ideas/hy- potheses on clarity, feasibility, novelty, plausibility, consistency, and impact. Exemples include ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">],AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark36"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">19</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark64"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AI-co-scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">47</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">], Sparks</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark56"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"> of Science [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">39</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark53"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Moose-chem [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">36</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Tong et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark46"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], PROTEUS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">29</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark74"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Rabby et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">57</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]. Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses an LLM judge to rank drug candidates by scientific rationale, pharmacological profile, and methodology of the supporting litera- ture. AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark23"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses a critic agent to assess the alignment between hypotheses and observational data. CLADD [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">6</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark28"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluates drugs by predicting its toxicity from molecule’s chemical representation(SMILES), infers likely protein targets (mechanism of action) from LLM agent generated comprehensive report on query molecule, and property categorization performance of molecules based on generated textual descrip- tion concatenated with its SMILES representation. DrugMCTS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">11</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] extends this by integrating LLM-based reward functions within a Monte Carlo Tree Search framework to score molecular–protein interactions using structural (binding pocket) and literature- derived contextual evidence. ChatBattery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark66"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses a ’Search’ Agent to verify novelty by querying the Materials Project database and a ’Retrieval’ Agent to identify similar lithium-based compounds from the ICSD, guiding subsequent hypothesis revisions. Liu et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">49</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluate materials design hypotheses based on the alignment between the stated materials design goals and the content of the hypothesis. LLM-Feynman </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark50"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">33</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark38"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employs an LLM to assign ’interpretability score’ to its own generated formulas, quantifying their physical and chemical meaning and ProtAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">21</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] assess success in generating proteins with desired secondary structures.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H2-S2: Literature-based assessment. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark54"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">To ensure that generated hypotheses are both novel and scientifically grounded, many systems incorporate literature-based assessment as a critical step. This process involves retrieving relevant prior work, evaluating semantic similarity, and determining whether the proposed idea contributes to new insights. Systems such as Scideator [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">37</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCIMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark52"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AI-scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">35</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AI-Scientist-v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AstroAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark51"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Graph of AI Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">34</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark36"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciAgents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">19</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Budget AI Researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Cheng et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark58"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Lee et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">41</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] deploy LLM-based ‘Novelty Checker’, which assesses the idea’s novelty against relevant papers. HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark78"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluates whether the rationale and hypothesis track the logic and chronology of literature chains. CtrlHGen [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">61</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] measures the alignment between hypothesis conclusions and the observed entities in the knowledge graph using multiple similarity metrics. These metrics are incorporated into the reward function that guides hypothesis generation.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H2-S3: Quantitative Assessment Using Domain Metrics. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark67"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Quantitative eval- uation using domain-specific scientific metrics and computational models provides an objective measure of hypothesis. dziner [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">50</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] calculate ’synthesizability’ (e.g. syn- thetic accessibility score, quantitative estimate of drug-likeness (QED score)) using PubChem-scale molecule dataset and specialized tools. The effectiveness of a proposed modification is then evaluated with domain-expert predictive models, and the LLM </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark40"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">ultimately decides whether to accept or reject the modification based on these quanti- tative results. Matexpert [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">23</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluates generated material structural validity, chemical feasibility and stability using quantitative and physics-informed metrics. ChatBat- tery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark37"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] verifies the stability and properties of the top-ranked cathode candidates using a machine learning surrogate model(MACE-MP) that predicts the energies and forces of the sampled structures. MAPPS [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">20</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark84"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluates crystal structures for struc- tural quality, thermodynamic stability, novelty, and accuracy. Liu et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">67</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark21"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] compare edited molecules with desired properties, quantify binding affinity for peptide editing and secondary structural content for protein secondary structure editing relative to desired targets. LLMatdesign [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">4</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark81"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] predicts properties via a Machine Learning Property Predictor (MLPP) after a Machine-Learning-Force-Field (MLFF) based relaxation of modified material structures, declaring success when predicted properties of the new material align with target specifications. In symbolic reasoning and equation discovery, DrSR [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">64</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark82"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and ChatSR [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">65</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark50"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] iteratively improve model output by assessing generated equations using numerical fitness scores and domain performance metrics. LLM-Feynman [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">33</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] provides a comprehensive quantitative evaluation framework using the Coefficient of Determination and Mean Absolute Error (MAE) for regression tasks, and accuracy, precision, recall, F1 score, and cross-entropy loss for classification-based symbolic modeling.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>H2-S4: Human evaluation.</strong> </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Expert review complements automated scoring with contextual judgment and domain nuance. Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark55"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Tong et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">38</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark45"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], AstroA- gents [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">28</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark42"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciPIP [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">25</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark43"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SciMON [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">26</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], CODESCIENTIST [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark19"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], SCI-IDEA [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">], Literature </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark63"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Meets Data [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">46</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], ChatBattery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark74"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Rabby et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">57</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark27"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and HypER [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">10</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] incorporate expert feedback to refine, rank, or filter hypotheses.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            &nbsp;
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>The Test Stage</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T1: Experimental Design Generation </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Experimental design generation translates top-ranked ideas and hypotheses into verifiable experimental plans, creating a systematic path to validate or refute scien- tific propositions. It comprises three common strategies: literature-informed design, LL-based design generation, and test execution(Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">C and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T1-S1: Literature-informed experimental design. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">This strategy leverages prior studies and domain references to ground experimental plans in established scientific practices while enabling innovation. Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark25"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] selects top-ranked ’in vitro’ models from the literature review to define therapeutic testing strategies and generates concise experimental outlines. ResearchAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">8</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark31"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] synthesizes problem information, methods, and existing designs from the literature into new protocols that emphasizing clarity, robustness, reproducibility, validity, and feasibility. Chain of Ideas [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">14</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark41"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses the few-shot prompting with examples of prior experiments, together with user-provided ideas and key entities, to guide design generation. Similarly, DrugAgent [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">24</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] constructs executable code from curated documentation, compiled from widely used libraries and models from the literature. Together, these approaches ensure the design evidence-based while adaptable to novel scientific contexts.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T1-S2: LLM-based experiment design generation. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Beyond literature grounding, systems increasingly rely on LLM reasoning to autonomously generate experimental designs, often producing executable workflows or code for validation. AI-Scientist- v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] employs ’Agentic Tree Search algorithm’ to systematically explore hypotheses during the experimentation phase. For each node in the experimental pathway, an LLM proposes a concrete experimentation plan and the Python code to implement it. Similarly, AI-researcher [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark80"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Lee et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">63</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark76"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Lui et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">59</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], Cheng et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark37"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] transform research analyses and development plans into executable code via LLM agents. In physics-based discovery, Zhou et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">20</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] invoke tools from a physics toolbox to translate workflow steps into executable, physically grounded Python code. CODESCIENTIST </span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">[</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark64"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses a ’planer agent’ that ingests the idea, expert comments, and a codeblock library to produce a concrete plan and required codeblocks. AI-co-scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">47</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark47"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] features a ’Generation Agent’ that proposes experimental protocols for downstream validations. In biomedical experimental setting, ProteinHypothesis [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">30</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark44"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] assesses the experimental feasibility of hypotheses and maps the hypotheses to laboratory techniques, while Chemist-X [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">27</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] generates control scripts for an agent to operate wet-lab platforms for validation.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>T1-S3: Test execution. </strong></b>This strategy validates LLM-generated hypotheses via <i style="margin:0px;padding:0px;text-indent:0px;">in silico </i></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">or wet-lab experiments, closing the discovery loop by determining the real-world feasibility and accuracy of these hypotheses. In Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], the experimental strategies are proposed for validating therapeutic candidates by identifying a relevant disease mechanism and <i style="margin:0px;padding:0px;text-indent:0px;">in vitro </i></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark86"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">models and executed by human researchers in an iterative lab- in-the-loop framework. In clinical hypothesis refinement, LA-CDM [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">69</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] uses a ’decision agent’ to request additional diagnostic tests to reduce uncertainty. Perovskite-R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] implements recommended protocols and fabricates complete solar cell devices to assess the efficacy of the selected additives. ChatBattery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark44"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] synthesizes and tests cathode candidates through wet-lab experiments. Chemist-X [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">27</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark73"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] validates generated reaction conditions through fully automated wet-lab experiments under LLM supervision. Abdel et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">56</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] test LLM-proposed drug combinations in the lab. </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark70"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Execution also includes computational validation. Tang et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">53</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark60"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] apply regression- based statistical models to test hypotheses about LLM’s explanatory power for societal outcomes. Genesys [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">43</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] evaluates the proposed language model design by verify- ing code correctness, conducting training-time experiments, and measuring success via a multi-scale fitness function. CODESCIENTIST [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] follows an iterative gener- ate–execute–reflect debugging loop, while AI-Scientist [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] and AI-Scientist-v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] test LLM-generated experiment prototypes by running associated Python code to test conceptual validity.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T2: Iterative refinement</strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark13"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"> Iterative refinement improves the quality and feasibility of hypotheses, ideas, and experimental designs by incorporating feedback from agents, data, and experts, ensuring responsive to new information and evaluations. There are three common strategies: agent feedback–guided refinement, data-driven refinement, and human feedback–guided refinement(Fig. </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">2</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark14"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">C and </span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">3</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T2-S1: Agent feedback–guided refinement. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark18"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Systems employ feedback loops in which LLM-based agents iteratively revise hypotheses, code, and designs to improve feasibility and correctness. AI-Scientist-v2 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">1</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] refines algorithm nodes through agent feedback by evaluating parent nodes’ outcome, where buggy parents trigger debugging using clind nodes, while non-buggy parents prompt experimental refinement. Key met- rics (e.g., loss values, training data) are logged and visualized and a Vision–Language Model (VLM) then critiques plots for label clarity, legend completeness, and data accuracy. Nodes flagged by the VLM are marked buggy, whereas validated nodes are retained, enabling continuous, feedback-driven improvement. Similarly, AI-researcher</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark35"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"> [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">18</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark86"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] utilizes an ’Advisor’ Agent to propose actionable refinement spanning implemen- tation details, validation studies, visualizations, and comparative analyses. In clinical settings, the ’Hypothesis’ Agent in LA-CDM [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">69</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] updates diagnoses and confidence estimates as patient-state information evolves, repeating until a final diagnosis is produced. T2-S2: Data-driven refinement. </span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark33"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Data-driven refinement keeps hypotheses, models, and experimental plans empirically grounded by updating them with new computational and experimental evidence. In Robin [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">16</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark44"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], human scientists upload raw or semi- processed data (e.g., flow cytometry, RNA-seq data), which are processed by the ’Finch’ analysis agent with the requested analysis to generate statistical tests, differential gene expression plots. Robin also distills actionable insights from the analysis results and proposes follow-up assays. Fully automated systems such as Chemist-X [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">27</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark73"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] execute LLM-generated reaction conditions with closed-loop wet-lab experiments and quantify reaction outcomes (e.g., the product yield using High-Performance Liquid Chromatography). Abdel et al. [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">56</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] validate LLM-generated drug combinations experimentally and compute Highest Single Agent (HSA) synergy scores to quantify performance. In materials science, ChatBattery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark57"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] characterizes physical/structural performance with power conversion efficiency, film quality, and stability. Perovskite- R1 [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">40</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark70"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">] assess the efficacy of the selected additives by fabricating complete solar cell devices and evaluating metrics such as Power Conversion Efficiency (PCE), film quality, and long-term device performance. In social science, Tang et al [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">53</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] convert natural language hypotheses into interpretable embeddings used as explanatory variables in a linear regression model to predict societal outcomes. Coefficients are tested for significance, non-significant hypotheses are pruned, and the feedback guides subsequent LLM proposals.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>T2-S2: Human feedback guided refinement. </strong></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark87"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Expert review adds domain-specific judgment that complements automated evaluations. In ChatBattery [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">70</span></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark26"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">], human researchers revise hypotheses after examining experimental outcomes, aligning hypothe- ses with objectives and constraints when results indicate limited validity or feasibility. CODESCIENTIST [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">9</span></span><span style="color:black;font-family:'Times New Roman', Times, serif;font-size:18px;"><span class="p" style="font-style:normal;font-weight:normal;margin:0pt;padding:0px;text-decoration:none;text-indent:0px;">] generates concise reports indicating whether each hypothe- sis was confirmed, rejected, or inconclusive, and final expert screening substantially narrows the candidate set.</span></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark15"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>‌</strong></span></a><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>End-to-End Workflow</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">The previous sections examined how individual studies address specific stages and substages of the EXHYTE cycle, providing a granular understanding of each method- ological contribution. However, this stage-by-stage analysis does not fully reveal how these strategies can be integrated into an end-to-end scientific discovery workflow. To illustrate how the surveyed components interact in practice, we present two compre- hensive systems: AI-Researcher and AI Co-Scientist. These case studies demonstrate how strategies from the EXHYTE framework combine to enable autonomous or semi- autonomous discovery, from exploration and hypothesis generation to testing and refinement.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark16"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>‌</strong></span></a><span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>AI-Researcher: an </strong><i style="margin:0px;padding:0px;text-indent:0px;"><strong>in silico </strong></i><strong>workflow</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">The goal of the AI-Researcher system is to design a complete research pipeline with minimal human intervention capable of systematically targeting the conceptual frontiers of science, deliberately seeking conceptual gaps, contradictory findings, and emerging patterns across literature and implementations. It begins with a curated collection of approximately 15–20 reference papers, along with optional datasets, that form the foundation of knowledge for the system. These sources mirror the intellectual basis of a target publication, providing the conceptual and methodological context from which new discoveries emerge.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Explore. </strong></b>During the exploration stage, AI-researcher establishes the technical and conceptual groundwork for autonomous algorithm design. This process is jointly man- aged by the <i style="margin:0px;padding:0px;text-indent:0px;">Knowledge Acquisition Agent </i>and the <i style="margin:0px;padding:0px;text-indent:0px;">Resource Analyst Agent</i>, supported by their respective sub-agents. The <i style="margin:0px;padding:0px;text-indent:0px;">Resource Analyst Agent </i>decomposes the research objective into atomic academic concepts, which are fundamental, indivisible elements requiring targeted investigation. Guided by these concepts, the <i style="margin:0px;padding:0px;text-indent:0px;">Knowledge Acquisition Agent </i>retrieves relevant papers and code repositories from scientific databases, applying multi-criteria filtering to select high-quality sources (E2-S1/S2). The <i style="margin:0px;padding:0px;text-indent:0px;">Paper Analyst </i>sub- agent extracts mathematical formulations from LaTeX files using a retrieval-augmented generation (RAG) approach (E3-S1), while the <i style="margin:0px;padding:0px;text-indent:0px;">Code Analyst </i>identifies corresponding code implementations, establishing explicit bidirectional mappings between theory and code (E3-S3). The resulting structured report provides the empirical and theoretical foundation for downstream reasoning.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Hypothesize. </strong></b>Building on the knowledge assembled during exploration, the <i style="margin:0px;padding:0px;text-indent:0px;">Idea Generator </i>identifies conceptual gaps, contradictory findings, and emerging patterns across literature and implementations (H1-S3). It employs a Divergent–Convergent Discovery Framework: the divergent phase produces multiple conceptually distinct research directions, while the convergent phase evaluates them using criteria such as novelty, soundness, and transformative potential (H2-S1/S2). The <i style="margin:0px;padding:0px;text-indent:0px;">Advisor Agent </i>provides expert-style feedback linking theoretical concepts to implementation plans, embodying the human-in-the-loop evaluation principle (H2-S4).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Test. </strong></b>The testing phase operationalizes selected hypotheses through experimental planning, implementation, and iterative refinement. The <i style="margin:0px;padding:0px;text-indent:0px;">Plan Agent </i>generates a com- prehensive development plan, including dataset selection, training/testing methodology, and evaluation metrics (T1-S1). The <i style="margin:0px;padding:0px;text-indent:0px;">Code Agent </i>translates this plan into executable implementations, verifying dataset usage and experimental consistency (T1-S2). Results are analyzed by the <i style="margin:0px;padding:0px;text-indent:0px;">Experiment Analysis Agent</i>, which recommends code modifications and further experimentation (T2-S1). Finally, the <i style="margin:0px;padding:0px;text-indent:0px;">Automated Documentation Agent </i>revises the manuscript using an academic checklist to ensure clarity, reproducibility, and completeness (T2-S3). Together, these stages demonstrate a continuous feedback loop linking hypothesis generation to empirical validation within a coherent <i style="margin:0px;padding:0px;text-indent:0px;">in silico </i>research pipeline.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><strong>AI Co-Scientist: a wet-lab–integrated workflow</strong></span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">The primary goal of the AI-scientist is to accelerate the process of scientific discovery by uncovering new, original knowledge operating within a "scientist-in-the-loop" collaborative paradigm. AI Co-Scientist implements the EXHYTE framework with physical experimentation by coupling LLM reasoning with wet-lab validation. The system accepts multiple input types from the human scientist, most critically a research goal, which may range from concise statements to long documents containing numerous prior publications.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Explore. </strong></b>Upon receiving the research goal, AI Co-Scientist parses it into a <i style="margin:0px;padding:0px;text-indent:0px;">research plan configuration </i>specifying desired outputs, constraints, and evaluation criteria (E1- S1). This configuration serves as a blueprint for downstream hypothesis generation. The <i style="margin:0px;padding:0px;text-indent:0px;">Generation Agent </i>then explores relevant literature via web search and retrieval tools (E2-S1/S2) to identify promising directions for novel findings.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Hypothesize. </strong></b>The <i style="margin:0px;padding:0px;text-indent:0px;">Generation Agent </i>grounds its reasoning in summarized prior work and produces hypotheses and research plans (H1-S3). A <i style="margin:0px;padding:0px;text-indent:0px;">Reflection Agent </i>simulates a scientific peer-review process, critically examining each hypothesis for correctness, novelty, and evidential grounding (H2-S2). Quick reviews are followed by comprehensive evaluations that integrate external search and evidence retrieval to assess validity. For complex hypotheses, deep verification decomposes each claim into constituent assumptions, which are independently examined to detect invalid components. The <i style="margin:0px;padding:0px;text-indent:0px;">Ranking Agent </i>employs an Elo-style tournament framework to compare hypotheses through multi-turn debates, scoring them on novelty, correctness, and testability (H2-S1). The top-ranked hypotheses are iteratively refined by an <i style="margin:0px;padding:0px;text-indent:0px;">Evolution Agent </i>(T2-S1/S3).</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;"><b style="margin:0px;padding:0px;text-indent:0px;"><strong>Test. </strong></b>The testing stage connects LLM-generated hypotheses to real-world validation. AI Co-Scientist has been deployed in several biomedical domains, demonstrating end-to- end discovery. Notably, the system generated the hypothesis that <i style="margin:0px;padding:0px;text-indent:0px;">chromosomal phage- inducible chromosomal islands (cf-PICIs) hijack phage tails to expand host range</i></span><a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" href="#bookmark88"><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">, which was experimentally validated [</span></a><span style="color:rgb(0,0,255);font-family:'Times New Roman', Times, serif;font-size:18px;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">71</span></span><span style="font-family:'Times New Roman', Times, serif;font-size:18px;">]. This “tail-hijacking” mechanism revealed a previously unknown route for horizontal gene transfer in bacteria, advancing understanding of antimicrobial resistance evolution. This case exemplifies how EXHYTE-integrated reasoning—spanning literature exploration, hypothesis generation, prioritization, and refinement—can produce experimentally verified discoveries.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:18px;">Together, AI-Researcher and AI Co-Scientist illustrate how EXHYTE strategies interconnect across modalities. In both, <i style="margin:0px;padding:0px;text-indent:0px;">Explore </i>stages ground the system in literature and data; <i style="margin:0px;padding:0px;text-indent:0px;">Hypothesize </i>stages generate and prioritize testable ideas; and <i style="margin:0px;padding:0px;text-indent:0px;">Test </i>stages operationalize and refine them through computation or experimentation. These end-to- end workflows demonstrate the practical synthesis of the modular strategies surveyed earlier, showing how autonomous systems can orchestrate the full scientific discovery process from exploration to empirical validation.</span>
        </p>
        <p style="margin-left:0pt;text-align:justify;">
            &nbsp;
        </p>
        <figure class="image" style="text-align: center; margin-top: 20px; margin-bottom: 20px;">
    <img src="image003.jpg" width="550" height="550" style="max-width: 100%; height: auto;">
</figure>
    </li>
</ol>
<p style="text-align:center;">
    <a style="-webkit-text-stroke-width:0px;color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-variant-caps:normal;font-weight:normal;letter-spacing:normal;margin:0px;orphans:auto;padding:0px;text-align:justify;text-decoration:none;text-indent:0px;text-transform:none;white-space:normal;widows:auto;word-spacing:0px;" name="bookmark14">‌</a><span style="color:rgb(0,0,0);font-family:&quot;Times New Roman&quot;, serif;font-size:10.666667px;"><span style="-webkit-text-stroke-width:0px;caret-color:rgb(0, 0, 0);display:inline !important;float:none;font-style:normal;font-variant-caps:normal;letter-spacing:normal;orphans:auto;text-align:justify;text-decoration:none;text-indent:0px;text-transform:none;white-space:normal;widows:auto;word-spacing:0px;font-size:medium"><strong>Fig. 3 </strong></span><span class="s2" style="-webkit-text-stroke-width:0px;caret-color:rgb(0, 0, 0);font-style:normal;font-variant-caps:normal;font-weight:normal;letter-spacing:normal;margin:0px;orphans:auto;padding:0px;text-align:justify;text-decoration:none;text-indent:0px;text-transform:none;white-space:normal;widows:auto;word-spacing:0px;font-size:medium;">AI methods for the EXHYTE Cycle and substages.</span></span>
</p>
        <p style="margin-left:0pt;text-align:justify;">
            <span style="font-family:'Times New Roman', Times, serif;font-size:22px;"><strong>References</strong></span>
        </p>
    </li>
    <li data-list-item-id="ebcd559db3428b67c34a085cc0bcdfe96">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:8pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            Yamada, Y., Lange, R.T., Lu, C., Hu, S., Lu, C., Foerster, J., Clune, J., Ha, D.: The ai scientist-v2: Workshop-level automated scientific discovery via agentic tree search. arXiv preprint arXiv:2504.08066 (2025)
        </p>
    </li>
    <li data-list-item-id="ea5e168d4107eddcfef2c2e0f9c074c66">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark19">‌</a>Keya, F., Rabby, G., Mitra, P., Vahdati, S., Auer, S., Jaradeh, Y.: Sci-idea: Context-aware scientific ideation using token and sentence embeddings. arXiv preprint arXiv:2503.19257 (2025)
        </p>
    </li>
    <li data-list-item-id="eb05f3fe8aa3033bbeece56be89714c39">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark20">‌</a>Garikaparthi, A., Patwardhan, M., Vig, L., Cohan, A.: Iris: Interactive research ideation system for accelerating scientific discovery. arXiv preprint arXiv:2504.16728 (2025)
        </p>
    </li>
    <li data-list-item-id="ef68cb44064b7ea25947d3216f3f5778d">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark21">‌</a>Jia, S., Zhang, C., Fung, V.: Llmatdesign: Autonomous materials discovery with large language models. arXiv preprint arXiv:2406.13163 (2024)
        </p>
    </li>
    <li data-list-item-id="e8850de40059755958e01028936cfacb3">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark22">‌</a>Yang, P., Zhao, C., Tang, B., Zhang, Z., Wang, X., Deng, Y., Lu, Y., Guan, C., Liu, Z., An, B.: Automat: A hierarchical framework for autonomous alloy discovery. arXiv preprint arXiv:2507.16005 (2025)
        </p>
    </li>
    <li data-list-item-id="e187b58d7591fec46db20e6c44686cd3a">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark23">‌</a>Lee, N., De Brouwer, E., Hajiramezanali, E., Biancalani, T., Park, C., Scalia, G.: Rag-enhanced collaborative llm agents for drug discovery. arXiv preprint arXiv:2502.17506 (2025)
        </p>
    </li>
    <li data-list-item-id="ed34b542ef4268eb46599aae43b6aea37">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark24">‌</a>Pu, Y., Lin, T., Chen, H.: Piflow: Principle-aware scientific discovery with multi- agent collaboration. arXiv preprint arXiv:2505.15047 (2025)
        </p>
    </li>
    <li data-list-item-id="eb9e3b2d97e185caed10d08310dcfc88f">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark25">‌</a>Baek, J., Jauhar, S.K., Cucerzan, S., Hwang, S.J.: Researchagent: Iterative research idea generation over scientific literature with large language models. arXiv preprint arXiv:2404.07738 (2024)
        </p>
    </li>
    <li data-list-item-id="ec8a25a588f3aba2e983b3f196358d760">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-15pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark26">‌</a>Jansen, P., Tafjord, O., Radensky, M., Siangliulue, P., Hope, T., Mishra, B.D., Majumder, B.P., Weld, D.S., Clark, P.: Codescientist: End-to-end semi- automated scientific discovery with code-based experimentation. arXiv preprint arXiv:2503.22708 (2025)
        </p>
    </li>
    <li data-list-item-id="e7949ba400a3a4e2077f4054e770fdd54">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark27">‌</a>Vasu, R., Basu, C., Mishra, B.D., Sarasua, C., Clark, P., Bernstein, A.: Hyper: Literature-grounded hypothesis generation and distillation with provenance. arXiv preprint arXiv:2506.12937 (2025)
        </p>
    </li>
    <li data-list-item-id="e96a899a7df700892ce5583d543260b09">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark28">‌</a>Yang, Z., Wan, Y., Li, Y., Matsuda, Y., Xie, T., Song, L.: Drugmcts: a drug repurposing framework combining multi-agent, rag and monte carlo tree search. arXiv preprint arXiv:2507.07426 (2025)
        </p>
    </li>
    <li data-list-item-id="e3943dc2f6799efabb306064abe40f367">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark29">‌</a> Knox, C., Wilson, M., Klinger, C.M., Franklin, M., Oler, E., Wilson, A., Pon, A., Cox, J., Chin, N.E., Strawbridge, S.A., <i style="margin:0px;padding:0px;text-indent:0px;">et al.</i>: Drugbank 6.0: the drugbank knowledgebase for 2024. Nucleic acids research <b style="margin:0px;padding:0px;text-indent:0px;"><strong>52</strong></b>(D1), 1265–1275 (2024)
        </p>
    </li>
    <li data-list-item-id="e8d9a0503af4b6cec962c669238c04cf1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark30">‌</a> Pahikkala, T., Airola, A., Pietilä, S., Shakyawar, S., Szwajda, A., Tang, J., Aittokallio, T.: Toward more realistic drug–target interaction predictions. Briefings in bioinformatics <b style="margin:0px;padding:0px;text-indent:0px;"><strong>16</strong></b>(2), 325–337 (2015)
        </p>
    </li>
    <li data-list-item-id="ebb9eeafda3111fb87e8772cc4ee48749">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark31">‌</a> Li, L., Xu, W., Guo, J., Zhao, R., Li, X., Yuan, Y., Zhang, B., Jiang, Y., Xin, Y., Dang, R., et al.: Chain of ideas: Revolutionizing research via novel idea development with llm agents. arXiv preprint arXiv:2410.13185 (2024)
        </p>
    </li>
    <li data-list-item-id="ebb3317bd9bf8fefaa1dd41b7b86e9b9d">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark32">‌</a>Deng, Y., Ericksen, S.S., Gitter, A.: Assay2mol: large language model-based drug design using bioassay context. arXiv preprint arXiv:2507.12574 (2025)
        </p>
    </li>
    <li data-list-item-id="ed5f3c8df9d668d9785b1ffefc27f817b">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 59pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark33">‌</a> Ghareeb, A.E., Chang, B., Mitchener, L., Yiu, A., Szostkiewicz, C.J., Laurent, J.M., Razzak, M.T., White, A.D., Hinks, M.M., Rodriques, S.G.: Robin: A multi- agent system for automating scientific discovery. arXiv preprint arXiv:2505.13400 (2025)
        </p>
    </li>
    <li data-list-item-id="ee71e6ca90f46afb034934d25f7b6bd12">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 59pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark34">‌</a> Liu, X., Guo, Y., Li, H., Liu, J., Huang, S., Ke, B., Lv, J.: Drugllm: Open large language model for few-shot molecule generation. arXiv preprint arXiv:2405.06690 (2024)
        </p>
    </li>
    <li data-list-item-id="e718d8cb3c2ea7e110040aab5beacd0fc">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark35">‌</a>Tang, J., Xia, L., Li, Z., Huang, C.: Ai-researcher: Autonomous scientific innovation. arXiv preprint arXiv:2505.18705 (2025)
        </p>
    </li>
    <li data-list-item-id="e142a6c8a46d6f5f65eba9f4da62c708c">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark36">‌</a>Ghafarollahi, A., Buehler, M.J.: Sciagents: automating scientific discovery through bioinspired multi-agent intelligent graph reasoning. Advanced Materials <b style="margin:0px;padding:0px;text-indent:0px;"><strong>37</strong></b>(22), 2413523 (2025)
        </p>
    </li>
    <li data-list-item-id="ec046ebb1f16c13ac3e2b067a9aa83746">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark37">‌</a>Zhou, L., Ling, H., Yan, K., Zhao, K., Qian, X., Arróyave, R., Qian, X., Ji, S.: Toward greater autonomy in materials discovery agents: Unifying planning, physics, and scientists. arXiv preprint arXiv:2506.05616 (2025)
        </p>
    </li>
    <li data-list-item-id="e40443b2bcebe422391b9bdae7600aaa4">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark38">‌</a>Ghafarollahi, A., Buehler, M.J.: Protagents: protein discovery via large language model multi-agent collaborations combining physics and machine learning. Digital Discovery <b style="margin:0px;padding:0px;text-indent:0px;"><strong>3</strong></b>(7), 1389–1409 (2024)
        </p>
    </li>
    <li data-list-item-id="e6c6fa889c663afd9297fd923804aedbf">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark39">‌</a>Ghafarollahi, A., Buehler, M.J.: Atomagents: Alloy design and discovery through physics-aware multi-modal multi-agent artificial intelligence. arXiv preprint arXiv:2407.10022 (2024)
        </p>
    </li>
    <li data-list-item-id="eb3abc67fe40c3f1bc90738800da2558c">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark40">‌</a>Ding, Q., Miret, S., Liu, B.: Matexpert: Decomposing materials discovery by mimicking human experts. arXiv preprint arXiv:2410.21317 (2024)
        </p>
    </li>
    <li data-list-item-id="edb54add8675b96f4b84a90c31797fe1c">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark41">‌</a>Liu, S., Lu, Y., Chen, S., Hu, X., Zhao, J., Lu, Y., Zhao, Y.: Drugagent: Automating ai-aided drug discovery programming through llm multi-agent collaboration. arXiv preprint arXiv:2411.15692 (2024)
        </p>
    </li>
    <li data-list-item-id="e3ca21f4114254e9b6faa9a38f8489012">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark42">‌</a> Wang, W., Gu, L., Zhang, L., Luo, Y., Dai, Y., Shen, C., Xie, L., Lin, B., He, X., Ye, J.: Scipip: An llm-based scientific paper idea proposer. arXiv preprint arXiv:2410.23166 (2024)
        </p>
    </li>
    <li data-list-item-id="ec5debe80c6384765b35143c4085405dc">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 35pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark43">‌</a> Wang, Q., Downey, D., Ji, H., Hope, T.: Scimon: Scientific inspiration machines optimized for novelty. In: Proceedings of the 62nd Annual Meeting of the Asso- ciation for Computational Linguistics (Volume 1: Long Papers), pp. 279–299 (2024)
        </p>
    </li>
    <li data-list-item-id="e65a9cc66d1aaa5af8e535127a22e2efe">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:left;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark44">‌</a>Chen, K., Li, J., Wang, K., Du, Y., Yu, J., Lu, J., Li, L., Qiu, J., Pan, J., Huang, Y.,
        </p>
    </li>
    <li data-list-item-id="e9191c391a244ee51908507c51bbe78a3">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 36pt;text-align:left;text-decoration:none;text-indent:0pt;">
            et al.: Chemist-x: Large language model-empowered agent for reaction condition recommendation in chemical synthesis. arXiv preprint arXiv:2311.10776 (2023)
        </p>
    </li>
    <li data-list-item-id="eb654a05460c108bfcc90cbde298bea85">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark45">‌</a>Saeedi, D., Buckner, D., Aponte, J.C., Aghazadeh, A.: Astroagents: A multi- agent ai for hypothesis generation from mass spectrometry data. arXiv preprint arXiv:2503.23170 (2025)
        </p>
    </li>
    <li data-list-item-id="ebbfa1a81e180485fc41abcf255444038">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark46">‌</a>Qu, S., Ding, N., Xie, L., Li, Y., Liu, Z., Zhang, K., Xiong, Y., Zuo, Y., Chen, Z., Hua, E., et al.: Automating exploratory multiomics research via language models. arXiv preprint arXiv:2506.07591 (2025)
        </p>
    </li>
    <li data-list-item-id="e70f52b800a8677fab584a0221e7357cd">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark47">‌</a> Bazgir, A., Zhang, Y., <i style="margin:0px;padding:0px;text-indent:0px;">et al.</i>: Proteinhypothesis: A physics-aware chain of multi- agent rag llm for hypothesis generation in protein science. In: Towards Agentic AI for Science: Hypothesis Generation, Comprehension, Quantification, and Validation (2025)
        </p>
    </li>
    <li data-list-item-id="ea9df426b4ef4e9e2513148c42eee48f1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" target="_blank" rel="noopener noreferrer" href="https://nomad-lab.eu/" name="bookmark48">Laboratory, N.: NOMAD Repository and Archive. </a><span style="color:rgb(0,0,255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;"><span style="font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;">https://nomad-lab.eu</span></span>. Accessed: 2025-11-20 (2025)
        </p>
    </li>
    <li data-list-item-id="e90197bea53710cbc0e35a8510535934f">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" target="_blank" rel="noopener noreferrer" href="https://mp-time-split.readthedocs.io/" name="bookmark49">Materials Project: Materials Project Time Splits and MPTS-52 dataset documen- tation (2024). </a><a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" target="_blank" rel="noopener noreferrer" href="https://mp-time-split.readthedocs.io/">https://mp-time-split.readthedocs.io/</a>
        </p>
    </li>
    <li data-list-item-id="ec382e076870011ea9aa197085e9e8994">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark50">‌</a>Song, Z., Ju, M., Ren, C., Li, Q., Li, C., Zhou, Q., Wang, J.: Llm-feynman: Leveraging large language models for universal scientific formula and theory discovery. arXiv preprint arXiv:2503.06512 (2025)
        </p>
    </li>
    <li data-list-item-id="e74276e46ac82e727ef2654168ad078da">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark51">‌</a>Gao, X., Zhang, Z., Xie, M., Liu, T., Fu, Y.: Graph of ai ideas: Leverag- ing knowledge graphs and llms for ai research idea generation. arXiv preprint arXiv:2503.08549 (2025)
        </p>
    </li>
    <li data-list-item-id="e5bdf518f09e247b5b70ccb35ba692a30">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 59pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark52">‌</a> Lu, C., Lu, C., Lange, R.T., Foerster, J., Clune, J., Ha, D.: The ai scientist: Towards fully automated open-ended scientific discovery. arXiv preprint arXiv:2408.06292 (2024)
        </p>
    </li>
    <li data-list-item-id="e8c22a6bb88c5e706b44ae767c35a97f4">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark53">‌</a>Yang, Z., Liu, W., Gao, B., Xie, T., Li, Y., Ouyang, W., Poria, S., Cambria, E., Zhou, D.: Moose-chem: Large language models for rediscovering unseen chemistry scientific hypotheses. arXiv preprint arXiv:2410.07076 (2024)
        </p>
    </li>
    <li data-list-item-id="e415d3222e18bdb271974401a0f9eb639">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark54">‌</a>Radensky, M., Shahid, S., Fok, R., Siangliulue, P., Hope, T., Weld, D.S.: Scideator: Human-llm scientific idea generation grounded in research-paper facet recombination. arXiv preprint arXiv:2409.14634 (2024)
        </p>
    </li>
    <li data-list-item-id="e6b6bd6d67a4a819a2c5e64f1e6cea2bf">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark55">‌</a>Tong, S., Mao, K., Huang, Z., Zhao, Y., Peng, K.: Automating psychological hypothesis generation with ai: when large language models meet causal graph. Humanities and Social Sciences Communications <b style="margin:0px;padding:0px;text-indent:0px;"><strong>11</strong></b>(1), 1–14 (2024)
        </p>
    </li>
    <li data-list-item-id="ecbbc1c579ca6d72226a925635dcf391e">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark56">‌</a>O’Neill, C., Ghosal, T., Răileanu, R., Walmsley, M., Bui, T., Schawinski, K., Ciucă, I.: Sparks of science: Hypothesis generation using structured paper data. arXiv preprint arXiv:2504.12976 (2025)
        </p>
    </li>
    <li data-list-item-id="e04829cef0a5920c75b2f2352fac16ea4">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:left;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark57">‌</a>Wang, X.-D., Chen, Z.-R., Guo, P.-J., Gao, Z.-F., Mu, C., Lu, Z.-Y.: Perovskite-
        </p>
    </li>
    <li data-list-item-id="ec703cebdb50e39863578edc94ed85874">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 61pt;text-align:left;text-decoration:none;text-indent:0pt;">
            r1: A domain-specialized llm for intelligent discovery of precursor additives and experimental design. arXiv preprint arXiv:2507.16307 (2025)
        </p>
    </li>
    <li data-list-item-id="e10fb1fd921dc186d810d2be5e2f36ece">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark58">‌</a>Lee, F., Ma, T.: The budget ai researcher and the power of rag chains. arXiv preprint arXiv:2506.12317 (2025)
        </p>
    </li>
    <li data-list-item-id="ee42c62dfe929d8617fd4a0aa0b8846ff">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 59pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark59">‌</a> Quilodrán-Casas, C., Waite, C., Alhadeff, N., Dsouza, D., Hughes, C., Kunstel- Tabet, L., Gilbert, A.: Towards unearthing neglected climate innovations from scientific literature using large language models. arXiv preprint arXiv:2411.10055 (2024)
        </p>
    </li>
    <li data-list-item-id="e6040a9d8b58e6eb7d63c6910642d0df1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark60">‌</a>Cheng, J., Clark, P., Richardson, K.: Language modeling by language models. arXiv preprint arXiv:2506.20249 (2025)
        </p>
    </li>
    <li data-list-item-id="eb1d2f1506ff4c9dd6c9082f9a07c5fa1">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark61">‌</a>Ciucă, I., Ting, Y.-S., Kruk, S., Iyer, K.: Harnessing the power of adversar- ial prompting and large language models for robust hypothesis generation in astronomy. arXiv preprint arXiv:2306.11648 (2023)
        </p>
    </li>
    <li data-list-item-id="e31f8f20b0e80cb0ff5f37e790026e519">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark62">‌</a>Sriram, A., Miller, B.K., Chen, R.T., Wood, B.M.: Flowllm: Flow matching for material generation with large language models as base distributions. Advances in Neural Information Processing Systems <b style="margin:0px;padding:0px;text-indent:0px;"><strong>37</strong></b>, 46025–46046 (2024)
        </p>
    </li>
    <li data-list-item-id="ee1296ffb05a4bb95d10d4915f978cb12">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark63">‌</a>Liu, H., Zhou, Y., Li, M., Yuan, C., Tan, C.: Literature meets data: A synergistic approach to hypothesis generation. arXiv preprint arXiv:2410.17309 (2024)
        </p>
    </li>
    <li data-list-item-id="edc536998c32b4cd59b97355449baa59d">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark64">‌</a> Gottweis, J., Weng, W.-H., Daryin, A., Tu, T., Palepu, A., Sirkovic, P., Myaskovsky, A., Weissenberger, F., Rong, K., Tanno, R., et al.: Towards an ai co-scientist. arXiv preprint arXiv:2502.18864 (2025)
        </p>
    </li>
    <li data-list-item-id="e718ce3231efd4282f1c3e230c8c58e14">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark65">‌</a>Chen, J., Zhang, K., Li, D., Feng, Y., Zhang, Y., Deng, B.: Structuring scientific innovation: A framework for modeling and discovering impactful knowledge combinations. arXiv preprint arXiv:2503.18865 (2025)
        </p>
    </li>
    <li data-list-item-id="ea316149044c52b51fc539c8f54ee19cd">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark66">‌</a>Liu, Q., Polak, M.P., Kim, S.Y., Shuvo, M.A.A., Deodhar, H.S., Han, J., Morgan, D., Oh, H.: Beyond designer’s knowledge: Generating materials design hypotheses via a large language model. Acta Materialia, 121307 (2025)
        </p>
    </li>
    <li data-list-item-id="eee777c587aad3e611ae2d421c09caff6">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark67">‌</a>Ansari, M., Watchorn, J., Brown, C.E., Brown, J.S.: dziner: Rational inverse design of materials with ai agents. arXiv preprint arXiv:2410.03963 (2024)
        </p>
    </li>
    <li data-list-item-id="ea3fbcde88ae889c98f3200d6bde74f6d">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:left;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark68">‌</a>Ma, T., Lin, X., Li, T., Li, C., Chen, L., Zhou, P., Cai, X., Yang, X., Zeng, D.,
        </p>
    </li>
    <li data-list-item-id="eb95268cf1a5021ad8a6d328b85209873">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:0px 0px 0px 36pt;text-align:left;text-decoration:none;text-indent:0pt;">
            Cao, D., et al.: Y-mol: A multiscale biomedical knowledge-guided large language model for drug development. arXiv preprint arXiv:2410.11550 (2024)
        </p>
    </li>
    <li data-list-item-id="ead3ccd59005ea5488909d893afa2de70">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark69">‌</a>Buehler, M.J.: Generative retrieval-augmented ontologic graph and multiagent strategies for interpretive large language model-based materials design. ACS Engineering Au <b style="margin:0px;padding:0px;text-indent:0px;"><strong>4</strong></b>(2), 241–277 (2024)
        </p>
    </li>
    <li data-list-item-id="e33ccf7fc00cf072227d54aa9c9464270">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark70">‌</a>Tang, Y., Qu, A., Yu, X., Deng, W., Ma, J., Zhao, J., Sun, L.: From street views to urban science: Discovering road safety factors with multimodal large language models. arXiv preprint arXiv:2506.02242 (2025)
        </p>
    </li>
    <li data-list-item-id="e8b0a7f2190dd50fe02228348da3542d5">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark71">‌</a>Yin, H., Kononova, A.V., Bäck, T., Stein, N.: Optimizing photonic structures with large language model driven algorithm discovery. In: Proceedings of the Genetic and Evolutionary Computation Conference Companion, pp. 2354–2362 (2025)
        </p>
    </li>
    <li data-list-item-id="eb1cd43ce3e6761d9f8982309329a5d4c">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark72">‌</a>Bystroński, M., Hołysz, M., Piotrowski, G., Chawla, N.V., Kajdanowicz, T.: Large language models as innovators: A framework to leverage latent space exploration for novelty discovery. arXiv preprint arXiv:2507.13874 (2025)
        </p>
    </li>
    <li data-list-item-id="e6b19d6fa1d136dc803b328c88e90ca3b">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark73">‌</a>Abdel-Rehim, A., Zenil, H., Orhobor, O., Fisher, M., Collins, R.J., Bourne, E., Fearnley, G.W., Tate, E., Smith, H.X., Soldatova, L.N., <i style="margin:0px;padding:0px;text-indent:0px;">et al.</i>: Scientific hypothesis generation by large language models: laboratory validation in breast cancer treatment. Journal of the Royal Society Interface <b style="margin:0px;padding:0px;text-indent:0px;"><strong>22</strong></b>(227), 20240674 (2025)
        </p>
    </li>
    <li data-list-item-id="e164eee50fc4d839790f4ae0d2e75a556">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark74">‌</a>Rabby, G., Muhammed, D., Mitra, P., Auer, S.: Iterative hypothesis generation for scientific discovery with monte carlo nash equilibrium self-refining trees. arXiv preprint arXiv:2503.19309 (2025)
        </p>
    </li>
    <li data-list-item-id="e70111d9938b0f291d82169856c2f93bf">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark75">‌</a>Guo, P., Zhang, Q., Lin, X.: Coevo: Continual evolution of symbolic solutions using large language models. arXiv preprint arXiv:2412.18890 (2024)
        </p>
    </li>
    <li data-list-item-id="e6b20f6d4b88a0fae94bfb9876087362e">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark76">‌</a>Liu, F., Tong, X., Yuan, M., Lin, X., Luo, F., Wang, Z., Lu, Z., Zhang, Q.: Evolution of heuristics: Towards efficient automatic algorithm design using large language model. arXiv preprint arXiv:2401.02051 (2024)
        </p>
    </li>
    <li data-list-item-id="e6eb339fa3ab8adf68ea29f7b95c53d42">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 59pt;text-align:justify;text-decoration:none;text-indent:-19pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark77">‌</a> Manning, B.S., Zhu, K., Horton, J.J.: Automated social science: Language models as scientist and subjects. Technical report, National Bureau of Economic Research (2024)
        </p>
    </li>
    <li data-list-item-id="e15c11a97fdd48f0746c4698f1e0488fd">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark78">‌</a>Gao, Y., Bai, J., Zheng, T., Sun, Q., Zhang, Z., Li, J., Song, Y., Fu, X.: Controllable logical hypothesis generation for abductive reasoning in knowledge graphs. arXiv preprint arXiv:2505.20948 (2025)
        </p>
    </li>
    <li data-list-item-id="ec2a2e6f859e52935d20e83b600090f25">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark79">‌</a>Zhou, Y., Liu, H., Srivastava, T., Mei, H., Tan, C.: Hypothesis generation with large language models. arXiv preprint arXiv:2404.04326 (2024)
        </p>
    </li>
    <li data-list-item-id="ed89ac1d37f8eb9cef98c001b1e7547c9">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark80">‌</a>Lee, K.-i., Koh, H., Lee, D., Yoon, S., Kim, M., Jung, K.: Generating diverse hypotheses for inductive reasoning. arXiv preprint arXiv:2412.13422 (2024)
        </p>
    </li>
    <li data-list-item-id="ed87cbd6a659c225540ec6738eb299624">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark81">‌</a>Wang, R., Wang, B., Li, K., Zhang, Y., Cheng, J.: Drsr: Llm based scientific equation discovery with dual reasoning from data and experience. arXiv preprint arXiv:2506.04282 (2025)
        </p>
    </li>
    <li data-list-item-id="e9bcb9342e05fab83c292b9af891a885e">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark82">‌</a>Li, Y., Li, W., Yu, L., Wu, M., Liu, J., Wei, S., Deng, Y.: Chatsr: Conversational symbolic regression
        </p>
    </li>
    <li data-list-item-id="e1fd7852fca1b2b39505691916121e574">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark83">‌</a>Zheng, Y., Koh, H.Y., Ju, J., Nguyen, A.T., May, L.T., Webb, G.I., Pan, S.: Large language models for scientific discovery in molecular property prediction. Nature Machine Intelligence, 1–11 (2025)
        </p>
    </li>
    <li data-list-item-id="ec636126308cc3b295f6897def19c7cec">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark84">‌</a>Liu, S., Wang, J., Yang, Y., Wang, C., Liu, L., Guo, H., Xiao, C.: Chatgpt-powered conversational drug editing using retrieval and domain feedback. arXiv preprint arXiv:2305.18090 (2023)
        </p>
    </li>
    <li data-list-item-id="e2a6d334a1d28c8dfb8f32ee94f26299e">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 60pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark85">‌</a>Zhu, Q., Luo, J.: Generative design ideation: a natural language generation approach. In: International Conference on-Design Computing and Cognition, pp. 39–50 (2022). Springer
        </p>
    </li>
    <li data-list-item-id="e3c2cf3c8fe75123c72504c8ccf13e036">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark86">‌</a>Bani-Harouni, D., Pellegrini, C., Özsoy, E., Keicher, M., Navab, N.: Language agents for hypothesis-driven clinical decision making with reinforcement learning. arXiv preprint arXiv:2506.13474 (2025)
        </p>
    </li>
    <li data-list-item-id="e52078fb27baa0a691de3905e1bc09a8e">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:10pt 0px 0px 61pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark87">‌</a>Liu, S., Xu, H., Ai, Y., Li, H., Bengio, Y., Guo, H.: Expert-guided llm reasoning for battery discovery: From ai-driven hypothesis to synthesis and characterization. arXiv preprint arXiv:2507.16110 (2025)
        </p>
    </li>
    <li data-list-item-id="ed10b17f5902b224c0ca53d6849584f45">
        <p style="color:black;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0pt;padding:4pt 0px 0px 36pt;text-align:justify;text-decoration:none;text-indent:-20pt;">
            <a style="color:rgb(0, 0, 255);font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;font-style:normal;font-weight:normal;margin:0px;padding:0px;text-decoration:none;text-indent:0px;" name="bookmark88">‌</a> Penadés, J.R., Gottweis, J., He, L., Patkowski, J.B., Daryin, A., Weng, W.-H., Tu, T., Palepu, A., Myaskovsky, A., Pawlosky, A., et al.: Ai mirrors experimental science to uncover a mechanism of gene transfer crucial to bacterial evolution. Cell (2025)
        </p>
    </li>
</ol>